lib/absinthe/pipeline.ex

```diff
@@ -48,6 +48,7 @@ defmodule Absinthe.Pipeline do
       {Phase.Parse, options},
       # Convert to Blueprint
       {Phase.Blueprint, options},
+      {Phase.Telemetry, event: [:absinthe, :resolution, :start]},
```

Comment:
We may want to specifically emit telemetry about the resolution phase. That does get tricky since resolution can run multiple times over the course of executing a document, but for example putting the start this far up the pipeline won't work with subscriptions which store the cached documents at a far later level of processing.

Reply:
I'm open to suggestions here. I could even go w/o the explicit `start` event, the minimum we need is simply to track the `start_time` to be able to calculate the `duration`

Reply:
What other telemetry about the resolution phase do you have in mind?